package.json

```diff
@@ -50,6 +50,7 @@
     "eslint-config-kentcdodds": "^12.2.1",
     "husky": "^0.13.3",
     "jest-cli": "^19.0.2",
+    "nps": "^5.0.5",
```

Comment:
I'm not sure when this happened, but this is needed for the build to be fixed :)